include/mbedtls/md.h

```diff
@@ -44,6 +44,14 @@
 extern "C" {
 #endif
 
+/**
+ * \brief     Enumeration of supported message digests
```

Comment:
This not touched in the recent documentation reworkings - @patater could you adapt it to the style guidelines?